### PR TITLE
SUS-3832: Remove broke WoWWiki skin

### PR DESCRIPTION
**Pull request by mszabo-wikia**
Remove ancient and broken `wowwiki` skin and related assets.

https://wikia-inc.atlassian.net/browse/SUS-3832

### CODE REVIEW

File: skins/wowwiki/IE50Fixes.css [hunk 1]
```diff
@@ -1,67 +0,0 @@
-/*
-** IE5.0 Fix Stylesheet
```
Comment (macbre) [inline skins/wowwiki/IE50Fixes.css:2]: :laughing:

**Review by macbre — APPROVED** [review]

File: extensions/wikia/SkinChooser/SkinChooser.class.php [hunk 2]
```diff
@@ -250,7 +250,7 @@ public static function onGetSkin( RequestContext $context, &$skin ) {
 				} else {
 					$userSkin = 'oasis';
 				}
-			} else if ( !empty( $wgAdminSkin ) && $userSkin != 'oasis' && $userSkin != 'monobook' && $userSkin != 'wowwiki' && $userSkin != 'lostbook' ) {
+			} else if ( !empty( $wgAdminSkin ) && $userSkin != 'oasis' && $userSkin != 'monobook' ) {
```
Comment (xkxd) [inline extensions/wikia/SkinChooser/SkinChooser.class.php:253]: why did you remove `lostbook` check?

**Review by xkxd — APPROVED** [review]

File: extensions/wikia/SkinChooser/SkinChooser.class.php (hunk 2 above)
Comment (mszabo-wikia) [reply extensions/wikia/SkinChooser/SkinChooser.class.php:253]: there's no such skin on our site, it refers to some product of a bygone era
